testsuite/tests/typing-layouts-unboxed-records/exhaustiveness.ml

```diff
@@ -34,7 +34,7 @@ Lines 2-4, characters 2-26:
 4 |   | #{ y = B; _ } -> false
 Warning 8 [partial-match]: this pattern-matching is not exhaustive.
 Here is an example of a case that is not matched:
-#{y=A; _ }
+#{y=A; x=B}
```

Comment:
Can you change the comment above this test to not be a CR and explain why the test is here?